docs/getting-started.md

```diff
@@ -1,6 +1,31 @@
-# Getting started
+# Welcome to pgroll's documentation
 
-This guide covers the key concepts of `pgroll` and how to get started with it.
+`pgroll` ensures 100% uptime during schema changes in your application. It is an open source command line interface for developers using [PostgreSQL](https://postgresql.org) as their data layer.
+
+## Getting started
+
+Learn how `pgroll` simplifies zero-downtime migrations for you
+
+* [Why use `pgroll`](why-use-pgroll)
```

Comment:
Inline code in list items looks odd (at least in the preview):

<img width="524" alt="image" src="https://github.com/user-attachments/assets/8dd15622-c130-4d31-8b86-ed40b904d0b6" />

I think we don't need to use backticks here.

```suggestion
* [Why use pgroll](why-use-pgroll)
```